perception/autoware_tensorrt_rtmdet/include/tensorrt_rtmdet/tensorrt_rtmdet_node.hpp

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// cspell: ignore RTMDET, rtmdet
+#ifndef TENSORRT_RTMDET__TENSORRT_RTMDET_NODE_HPP_
+#define TENSORRT_RTMDET__TENSORRT_RTMDET_NODE_HPP_
+
+#include "tensorrt_rtmdet/tensorrt_rtmdet.hpp"
+
+#include <autoware/universe_utils/ros/debug_publisher.hpp>
+#include <autoware/universe_utils/system/stop_watch.hpp>
+#include <image_transport/image_transport.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include "autoware_internal_msgs/msg/segmentation_mask.hpp"
+#include "autoware_perception_msgs/msg/object_classification.hpp"
+#include "tier4_perception_msgs/msg/detected_object_with_feature.hpp"
+#include "tier4_perception_msgs/msg/detected_objects_with_feature.hpp"
+#include "tier4_perception_msgs/msg/feature.hpp"
```

Comment:
`feature` is not needed.
`detected_object_with_feature` and `object_classification` should be placed in source file.